BraintreeCore/src/main/java/com/braintreepayments/api/core/PaymentMethod.kt

```diff
@@ -25,79 +30,20 @@ abstract class PaymentMethod {
         private const val DEFAULT_INTEGRATION = "custom"
     }
 
-    private var _sessionId: String? = null
-    private var _source: String? = DEFAULT_SOURCE
-    private var _integration: String? = DEFAULT_INTEGRATION
-
-    abstract val apiPath: String?
-
-    /**
-     * @suppress
-     */
-    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-    constructor()
-
-    /**
-     * Sets the integration method associated with the tokenization call for analytics use.
-     * Defaults to custom and does not need to ever be set.
-     *
-     * @param integration the current integration style.
-     * @suppress
-     */
```

Comment:
We'll want to move these docstrings into property annotations ([similar to here](https://github.com/braintree/braintree_android/blob/f13626a796ae46b21fe41d62b0092c7f73d3c036/BraintreeCore/src/main/java/com/braintreepayments/api/core/Authorization.kt#L8)).

Reply:
I will rollback PaymentMethod and remove super classes from Card.